dmsite/venv/src/SearchResultsPage/SearchResultsPage.jsx

```diff
@@ -1,16 +1,76 @@
 import React from 'react';
-import { Container, ListGroup, ListGroupItem } from 'reactstrap';
+import _ from 'lodash';
```

Comment:
what is lodash?

Reply:
JavaScript utility library delivering modularity, performance & extras.
https://lodash.com/